apiextensions/v1alpha1/doc.go

```diff
@@ -0,0 +1,9 @@
+// +k8s:deepcopy-gen=package,register
+// +k8s:defaulter-gen=TypeMeta
+// +k8s:openapi-gen=true
+// +openshift:featuregated-schema-gen=true
+
+// +kubebuilder:validation:Optional
```

Comment:
Do not do this please, even though each field should be explicitly marked, we don't want to influence the package for unmarked fields anyway